src/components/previewCard/index.module.css

```diff
@@ -0,0 +1,3 @@
+.promotionImage {
+    height: 209px;
+  }

```

Comment:
some inconsistent formatting here. indent with 2 spaces and don't indent the closing bracket (not sure why the style linter does not help you with that)